src/notebook/notebookKernel.ts

```diff
@@ -64,8 +64,7 @@ export class JuliaKernel {
         const execution = this.controller.createNotebookCellExecution(cell)
         execution.executionOrder = executionOrder
 
-        // TODO For some reason this doesn't work here
-        // await execution.clearOutput()
+        await execution.clearOutput()
```

Comment:
Clearing of output via the Execution task can only be done after you start the execution task.
In this case its best to create a whole new execution task prior to queueing of the cells.

I.e. i'd change as follows:
* Create an execution task, start it, clear the output & then end that execution task (we're now done with this execution task).
* Then create another new execution task & push into the `scheduledExecutionTasks`.

Reply:
Ah, yes, that works, I just pushed that.

It is not ideal, though :) Now when one queues a large notebook, for all cells the stopwatch "running" status icon briefly appears, then the output disappears, and then the status changes to the queued state. Maybe the underlying code could eventually allow calls to `clearOutput` _before_ one even starts execution? I'll merge this for now, though.

I also changed when an execution order gets assigned, namely when execution of a cell starts, not when it is queued. That brings things more in line with other clients.

And I changed cancel token handling a bit, so that queued cells that get cancelled immediately leave the queued state.